src/components/truncated-text/index.tsx

```diff
@@ -19,14 +19,16 @@ interface Props {
   lineHeight: number;
   entityTheme?: Entity;
   onlyTruncate?: boolean;
+  setTruncateTrue?: boolean;
```

Comment:
Anbefaler å følge en navnekonvensjon, da blir det veldig lesbart i koden. "setXXX" kan gjerne brukes for funksjoner, mens dette er en boolean. Jeg ville f.eks. kalt denne "isTruncate".